### DISCUSSION TITLE
CA and KRA

### DISCUSSION THREAD

**Discussion by Nourelkhatib** [dogtagpki/pki#4712] [Q&A]
I try to use profile "Manual User Dual-Use Certificate Enrollment using server-side Key generation" but I get this error "KRA Transport Certificate needs to be imported into the CA nssdb for Server-Side Kegen Enrollment". I search about it and which I found that there are no need to import the transport certificate if I install kra and ca in the same instance tomcat . But I still got that error even my ca and kra are in the same pki-tomcat instance . Can any one help me Please

**Comment by rriemann** [comment]
I found this link: https://docs.redhat.com/en/documentation/red_hat_certificate_system/9/html/planning_installation_and_deployment_guide/configuration_for_server-side_keygen#server-side_keygen_installation_configuration on this topic
